.github/workflows/pr-check.yaml

```diff
@@ -75,3 +75,7 @@ jobs:
 
       - name: Run tests
         run: yarn test
+
+      - name: Run build
+        if: ${{ matrix.os=='ubuntu-22.04' }}
```

Comment:
```suggestion
```

why do not build on each platform ?

you may have developers trying to build the extension on other platforms than Linux

Reply:
The goal for now was just to do everything from next-build, so when Windows build here failed with a generic and un-actionable message I reverted to only the platform next-build is on (ubuntu).

I just tried reverting that change and it built ok on windows twice, so hoping it was just a rare github/windows/yarn fluke and going with all three platforms.